src/app/info/page.tsx

```diff
@@ -0,0 +1,305 @@
+import { lang } from 'utils/language';
+import PositionInfobox from 'components/corps/position-infobox';
+import { api } from 'trpc/server';
+import { IconMail } from '@tabler/icons-react';
+import ActionIcon from 'components/input/action-icon';
+
+
+const styrelseOrder: Dictionary<number> = {
+  "Ordförande": 1,
+  "ViceOrdförande": 2,
+  "Sekreterare": 3,
+  "Kassör": 4,
+};
+
+
+const Positions = async () => {
+  const roles = await api.permission.getRoles.query();
+  const ordoredBoardCorps = await Promise.all(
+    roles
+      .filter(role => Object.keys(styrelseOrder).includes(role.name))
+      .sort((a, b) => styrelseOrder[a.name] - styrelseOrder[b.name])
+      .flatMap(role =>
+        role.corpsii.map(async (corps) => {
+          const result = await api.corps.get.query({ id: corps.id });
+          if (!result) throw new Error("Corps not found");
+          return result;
+        })
+      )
+  );
+
+  const TrivselCorps = await Promise.all(
+    roles
+      .filter(role => role.name === "Trivselombud")
+      .flatMap(role =>
+        role.corpsii.map(async (corps) => {
+          const result = await api.corps.get.query({ id: corps.id });
+          if (!result) throw new Error("Corps not found");
+          return result;
+        })
+      )
+  );
+
+  return (
+    <div>
+      <h2>{lang('Ansvarsposter', 'Positions of responsibility')}</h2>
+      <div className='max-w-4xl'>
+        <div className='flex flex-col space-y-2 p-4'>
+          <h3>{lang('Styrelsen', 'The Board')}</h3>
+          {lang(`Styrelsen leder Bleckhornen under ett verksamhetsår. Tillsammans med dirigenter,
+          balettledare, utskottsmedlemmar och andra förtroendevalda ansvarar styrelsen för planering
+          av repor, spelningar, sittningar, konserter, resor och andra aktiviteter. Detta görs med stöd och
+          samarbete med hela corpset`,
```

Comment:
In general, we should decide which terms should be capitalized or not, so the use remains consistent. Right now it is a bit mixed.